### ISSUE TITLE
Planning add support for nfdump 1.7.x version?

### ISSUE THREAD

**Issue by MuriloChianfa** [netx-as/libnf#10]
Hello guys, im receiving this error when using the latest version of Net::NfDump on pearl for read nfcapd files generated by nfdump 1.7.3:

![image](https://github.com/netx-as/libnf/assets/60560085/b88057b0-491b-43d6-a620-f987b42724aa)

Net::NfDump:  can not open/read file /netflow/demo/rt-demo/2024/01/19/nfcapd.202401191840 at /tmp/par-726f6f74/cache-1f28ba64695b5a1626038d153d8e8a1be49eea49/inc/lib/Net/NfDump.pm line 646.

Im suspecting of the new nfdump fields such as src_geo and dst_geo on 1.7.x versions.

PS. for nfcapd files generated by nfdump 1.6.x works fine.

Anyone can help me?

**Comment by mgregr** [comment]
Yes, the current release of libnf doesn't support the new file format of nfdump 1.7 yet. There is a preliminary work in libnf 1.7 branch, but it's not finished yet. Hopefully we will add the support for the new file format in future.
